### ISSUE TITLE
dat.GUI style constraint shorthand

### ISSUE THREAD

**Issue by edankwan** [wearekuva/oui#8]
At the moment, adding constraints to the dat.GUI style item is pretty redundant
```js
var gui = oui.datoui();
oui.annotate({min: 0, max: 5})(gui.add( settings, 'speed'), 'speed');
```

I would recommend using an optional third argument which is closer to the dat.GUI api like this:
```js
var gui = oui.datoui();
gui.add( settings, 'speed', {min: 0, max: 5} );
```

**Comment by marklundin** [comment]
Yep agreed. Constraints object shoulde be the same format as those passed to annotate. This will also support other controllers

**Comment by marklundin** [comment]
`datoui.add` now accepts a constraint object as a third param. This is the same as would be passed to `@annotate`.

Fixed in v0.0.14
